src/transforms/add_fields.rs

```diff
@@ -15,6 +15,8 @@ use toml::value::Value as TomlValue;
 #[serde(deny_unknown_fields)]
 pub struct AddFieldsConfig {
     pub fields: Fields<TomlValue>,
+    #[serde(default = "crate::serde::default_true")]
+    pub r#override: bool,
```

Comment:
might be cleaner to `#[serde(rename = "override")]` then name this field something that isn't a reserved keyword.